Source/SwiftLintCore/Rules/SuperfluousDisableCommandRule.swift

```diff
@@ -12,6 +12,16 @@ public struct SuperfluousDisableCommandRule: ConfigurationProviderRule, SourceKi
             in the disabled region. Use " - " if you wish to document a command.
             """,
         kind: .lint
+        nonTriggeringExamples: [
+            Example("let abc:Void // swiftlint:disable:this colon"),
+            Example("// swiftlint:disable colon (If it is enabled in the configuration file"),
+            Example("// swiftlint:enable colon (If it is disabled in the configuration file")
```

Comment:
I wonder how the example can at the same time trigger and not trigger the rule. 🤔

Reply:
It depends on the configuration, if it's already enabled/disabled in the configuration, the enable/disable command doesn't do anything and is therefore linted. I haven't tested this personally but that's how I imagined it'd work.

Reply:
I went ahead and tested it manually, that is indeed the case.